src/Dashboard/Dashboard.js

```diff
@@ -24,6 +24,7 @@ const Dashboard = (props) => {
     scenarioId,
     csmSimulationRun,
     scenarioState,
+    downloadScenarioRunCumulatedLogsFile,
```

Comment:
Could we use a shorter name for this function in Dashboard and DashboardPlaceholder ? "downloadLogs" seems understandable enough to me :)

Reply:
Done